Index: src/main/java/tier2/tier2/Tier2Application.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package tier2.tier2;\r\n\r\nimport org.springframework.boot.SpringApplication;\r\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\r\n\r\n@SpringBootApplication\r\npublic class Tier2Application {\r\n\r\n\tpublic static void main(String[] args) {\r\n\t\tSpringApplication.run(Tier2Application.class, args);\r\n\t}\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/tier2/tier2/Tier2Application.java	(revision 9256707742fba807c43d0244073d7bd27d5ee297)
+++ src/main/java/tier2/tier2/Tier2Application.java	(date 1638183714910)
@@ -8,6 +8,7 @@
 
 	public static void main(String[] args) {
 		SpringApplication.run(Tier2Application.class, args);
+		System.out.println("Hello");
 	}
 
 }
